public/fullSchema.json

```diff
@@ -1,7 +1,7 @@
 {
   "openapi": "3.0.0",
   "info": {
-    "version": "2.2.2",
+    "version": "2.3.0",
```

Comment:
why `fullSchema` is not regenerated?

Reply:
@damonspencerroberts could you instruct @jinglitw how we should update schema now?